### DISCUSSION TITLE
Defining the Train / Eval split in the config

### DISCUSSION THREAD

**Discussion by oliviercwa** [explosion/spaCy#9049] [Help: Coding & Implementations] [answered]
I am training a NER with a set of examples stored in a DocBin.

I am defining the training data and eval data set paths in the config using

```
[paths]
train = "ner-training-data.spacy"
dev = "ner-eval-data.spacy"
```

I was wondering if there is a way to use a single data set and define the train/eval split in the configuration ? Something like
```
[paths]
dataset = "ner-data.spacy"

[training]
train_split = 0.7
train_eval = 0.3
```

**Comment by polm** [comment, marked as answer]
No, we haven't added that functionality.

**Reply by polm** [reply]
Note that you can see examples of splitting data in sample projects, like [here](https://github.com/explosion/projects/blob/9d5fce5f95ddf5f35c3370b2074b25e995525f51/pipelines/ner_wikiner/scripts/partition.py) and [here](https://github.com/explosion/projects/blob/9e8c0a317f32f3004b26462166c92bdb6e5a7b37/tutorials/ner_pytorch_medical/scripts/preprocess.py).
